Podfile

```diff
@@ -173,8 +173,12 @@ abstract_target 'Apps' do
     ##
     pod 'Charts', '~> 3.2.2'
     pod 'Gifu', '3.2.0'
-    pod 'AppCenter', '4.1.1', :configurations => ['Release-Internal', 'Release-Alpha']
-    pod 'AppCenter/Distribute', '4.1.1', :configurations => ['Release-Internal', 'Release-Alpha']
+
+    app_center_version = '~> 4.1'
```

Comment:
I think it makes sense to use this because we want to be up-to-date with new features and fixes, but want to be intentional about updating if there's going to be a breaking change.

Feel free to push back if you see this dependency as something we should update only when a new feature we need demands it.